app/helpers/offender_helper.rb

```diff
@@ -18,9 +18,9 @@ def pom_responsibility_label(offender)
 
   def case_owner_label(offender)
     if offender.pom_responsible?
-      'Custody'
+      'Responsible'
     elsif offender.com_responsible?
-      'Community'
```

Comment:
I believe this change has an impact on the display of `_case_information.html.erb`, picture below. If this has been signed off by content, then great, otherwise it may need review on whether to either change the name of the field or to create a new method that returns 'Responsible' or 'Supporting' while leaving `case_owner_label` unchanged
<img width="1203" alt="image" src="https://user-images.githubusercontent.com/102144566/190156033-9c74ac55-395c-4b02-a813-7ade4390bd0c.png">